src/main/java/com/aws/greengrass/componentmanager/ClientConfigurationUtils.java

```diff
@@ -117,12 +121,15 @@ private static void configureClientMutualTLS(ApacheHttpClient.Builder httpBuilde
         TrustManager[] trustManagers = createTrustManagers(rootCAPath);
         KeyManager[] keyManagers = deviceConfiguration.getDeviceIdentityKeyManagers();
 
-        httpBuilder.tlsKeyManagersProvider(() -> keyManagers).tlsTrustManagersProvider(() -> trustManagers);
+        httpBuilder
+                .tlsKeyManagersProvider(() -> keyManagers)
+                .tlsTrustManagersProvider(() -> trustManagers);
     }
 
     private static TrustManager[] createTrustManagers(String rootCAPath) throws TLSAuthException {
         try {
-            List<X509Certificate> trustCertificates = EncryptionUtils.loadX509Certificates(Paths.get(rootCAPath));
+            List<X509Certificate> trustCertificates =
+                    EncryptionUtils.loadX509Certificates(Paths.get(rootCAPath));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Path traversal vulnerability detected. User-controlled input in file paths
can allow attackers to access files outside intended directories using
`../` sequences. Secure your code by using framework functions like
`getCanonicalPath()`, `normalize()`, or validating paths with
`.startsWith()` checks. Learn more:
https://cwe.mitre.org/data/definitions/22.html